brvs-core/src/main/java/iroha/validation/transactions/provider/impl/BasicTransactionProvider.java

```diff
@@ -190,7 +192,8 @@ private void processCommitted(List<Transaction> blockTransactions) {
               registerCreatedAccountByTransactionScanning(transaction);
               modifyUserQuorumIfNeeded(transaction);
             } catch (Exception e) {
-              logger.warn("Couldn't process account changes from the committed block", e);
+              logger.error("Couldn't process account changes from the committed block", e);
```

Comment:
I don't think that this is a good idea to log and throw. If you throw an exception, we expect you to log it on the caller side.